tests/python/pants_test/rules/test_test_integration.py

```diff
@@ -81,6 +81,7 @@ def test_passing_python_test(self):
     )
 
   def test_failing_python_test(self):
+    self.maxDiff = None
```

Comment:
What's this for? Seems unrelated and probably better to break out to separate PR.

Reply:
This is one of the tests that was failing and this is an unintentional addition from when I was diagnosing the issue.